quickwit-cli/tests/cli.rs

```diff
@@ -32,6 +32,7 @@ use quickwit_common::rand::append_random_suffix;
 use quickwit_common::uri::Uri;
 use quickwit_config::CLI_INGEST_SOURCE_ID;
 use quickwit_core::get_cache_directory_path;
+use quickwit_indexing::actors::INDEXING;
```

Comment:
I initially wondered what this was: `INDEXING_DIR_NAME` might be a better name. I bet you I came up with that not so great name in the first place :D